pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -138,6 +139,7 @@ func NewStaticPolicy(topology *topology.CPUTopology, numReservedCPUs int, reserv
 	}
 
 	klog.InfoS("Static policy created with configuration", "options", opts)
+	klog.InfoS("Static policy created with configuration", "llcAlignment", llcalign.IsEnabled())
```

Comment:
This log jumps out at me a little bit. Maybe we make the log more specific to llcalign?

Reply:
yeah, fair point. I'll try something.